lib/ffi/src/lib.rs

```diff
@@ -192,7 +192,7 @@ pub extern "C" fn rv_init(desc: InitDescriptor) -> Option<ptr::NonNull<Context>>
             } else {
                 desired_height
             },
-            ..wgpu::Limits::downlevel_webgl2_defaults()
```

Comment:
let's keep this webgl2 stuff, just add `max_storage_buffers_per_shader_stage: 1` initialization field